packages/cli/src/commands/init.ts

```diff
@@ -33,20 +51,68 @@ async function gitHubOAuth(): Promise<string> {
     return await serverResponse(httpServer);
   } catch (error) {
     // eslint-disable-next-line no-console
-    console.log(error);
+    console.error(error);
     return null;
   } finally {
     httpServer.close();
   }
+}
 
-  function serverResponse(server: HttpServer): Promise<string> {
-    return new Promise((resolve) => {
-      const interval = setInterval(() => {
-        if (server.token) {
-          clearInterval(interval);
-          resolve(server.token);
-        }
-      }, 300);
-    });
-  }
+function serverResponse(server: HttpServer): Promise<string> {
+  return new Promise((resolve) => {
+    const interval = setInterval(() => {
+      if (server.token) {
+        clearInterval(interval);
+        resolve(server.token);
+      }
+    }, 300);
+  });
+}
+
+async function createOrganization(config: Configstore, organizationName: string): Promise<IOrganizationDTO> {
+  return (
+    await axios.post(
+      API_CREATE_ORGANIZATION_URL,
+      { name: organizationName },
+      {
+        headers: {
+          authorization: `Bearer ${config.get('token')}`,
```

Comment:
There is a way in axios to add `default` headers so that you don't have to copy this line of code for each request. They will be attached automatically for every request. You can set the default headers after you have the JWT token.

Reply:
I like the sound of default headers I will look it up. ❤️